### PR TITLE
Configure Styling Component

### PR DESCRIPTION
**Pull request by alex-latham**
# Configure Styling Component

Please include a summary of the change:
* Adds styling to the configure component at solarizer.herokuapp.com/configure. Includes a near-transparent card containing a form to configure a potential solar array on top of a sky background. Items in the form such as title, inputs, and submit button mirror the styling established on the home page address form.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
Note any questions or struggles as well.
* Phase 3, struggles included preventing the width of the inputs from becoming too large -- solved by applying the max-width CSS property to the form container.

## Type of change
Please uncheck options that are not relevant.
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Add Testing
- [X] Refactor

## Checklist:
- [X] My code follows the Turing style guidelines
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/containers/Configure/Configure.js [hunk 1]
```diff
@@ -52,102 +52,104 @@ const Configure = ({ userQuote, allUserQuotes }) => {
 
   return (
     <>
-      {formCompleted && <Redirect push to="/historical" />}
+      {formCompleted && <Redirect to="/historical"/>}
```
Comment (J-Poulter) [inline src/containers/Configure/Configure.js:55]: If the url history doesn't work when using the back button, we can add the `push` attribute inside the Redirect and that should fix it

File: src/containers/Configure/Configure.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
-.configure-container {
+.background-container {
+  justify-content: center;
+  align-items: center;
   display: flex;
-  flex-direction: column;
   background-image: linear-gradient(
```
Comment (J-Poulter) [inline src/containers/Configure/Configure.css:5]: I think we should actually move this background property to a div container on the App component.  That way we don't have to reproduce the rule for each individual page

File: src/containers/Configure/Configure.js (hunk 1 above)
Comment (alex-latham) [reply src/containers/Configure/Configure.js:55]: Roger

File: src/containers/Configure/Configure.css (hunk 2 above)
Comment (alex-latham) [reply src/containers/Configure/Configure.css:5]: I'll do that now, thanks! Keeping our code DRY

File: src/containers/Configure/Configure.js (hunk 1 above)
Comment (nickstaylor) [reply src/containers/Configure/Configure.js:55]: the push idea works in this instance

**Comment by nickstaylor** [comment]
Good work Alex!  This PR closes #54 .

**Pull request merged**